source/includes/steps-generated-models-ui.yaml

```
@@ -0,0 +1,37 @@
+title: Navigate to the SDKs Screen
+ref: navigate-to-the-sdk-screen
+content: |
+  To begin, navigate to the
+  :guilabel:`SDKs` screen under the Build section in the {+leftnav+}.
+  Here you will find the tools to make your app compatible with different SDKs.
+
+---
+title: Select the Data Models Tab
+ref: select-the-data-models-tab
+content: |
+  From the Data Models tab, you can view the data object model for your
+  SDK.
+
+---
+title: Select Your Data Model Language 
+ref: select-your-data-model-language
+content: |
+  Under Realm Data Models, choose which language you are using for development. 
+  You will be shown the translasted version of each of your JSON data schemas, to whatever 
```

```suggestion
  The page contains a generated version of each of your object types translated to the
```

^ We typically avoid future tense